File: Carts/Controllers/CABaseViewController.m
//
//  CABaseViewController.m
//  GroceryList
//
//  Created by Hakon Hanesand on 3/4/15.

#import "CABaseViewController.h"

@implementation CABaseViewController

- (CAAnimationStack *)animationStack {
    if (!_animationStack) {
        _animationStack = [[CAAnimationStack alloc] init];
    }
    
    return _animationStack;
}

@end

File: Carts/Controllers/CABaseViewController.swift
//
//  CABaseViewController.swift
//  Carts
//
//  Created by Hakon Hanesand on 4/26/15.
//  Copyright (c) 2015 Hakon Hanesand. All rights reserved.

import UIKit

class CABaseViewController: UIViewController {
    
    lazy var animationStack: CAAnimationStack = {
        return CAAnimationStack()
    }()
}
